stop.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/weaveworks/footloose/pkg/cluster"
+)
+
+var stopCmd = &cobra.Command{
+	Use:   "stop",
+	Short: "Stop cluster machines",
+	RunE:  stop,
+}
+
+var stopOptions struct {
+	config string
+}
+
+func init() {
+	stopCmd.Flags().StringVarP(&stopOptions.config, "config", "c", Footloose, "Cluster configuration file")
+	footloose.AddCommand(stopCmd)
+}
+
+func stop(cmd *cobra.Command, args []string) error {
```

Comment:
Could this be used to only stop an individual machine? I could see a reason to stop and start a single machine instead the whole cluster.

Reply:
That sounds like a good idea. I opened https://github.com/weaveworks/footloose/issues/108 to track the idea. I don't think we need to gate this PR on having it implemented.